tests/tape/tapes/test_qubit_param_shift.py

```diff
@@ -46,13 +46,13 @@ def test_non_differentiable(self):
         assert tape._par_info[2]["grad_method"] == "A"
 
     def test_independent(self):
-        """Test that an independent variable is properly marked
+        """Test that an independent qml.variable is properly marked

```

Comment:
From hereon in I'm giving up on qml.variable, but I'll look out for qml.expval as that's harder

Reply:
```suggestion
        """Test that an independent variable is properly marked
```

Reply:
In my PhD thesis, I had a typo; I meant to say 'hereon' but accidentally typed 'heron'. One of my PhD examiners left a comment 'I don't see what this section has to do with the aquatic bird'.